native_client/README.md

```diff
@@ -18,18 +18,12 @@ ln -s ../DeepSpeech/native_client ./
 
 ## Building
 
-Before building the TensorFlow stand-alone library, you will need to prepare your environment to configure and build TensorFlow. Follow the [instructions](https://www.tensorflow.org/install/install_sources) on the TensorFlow site for your platform, up to the end of 'Configure the installation'.
+Before building the DeepSpeech client libraries, you will need to prepare your environment to configure and build TensorFlow. Follow the [instructions](https://www.tensorflow.org/install/install_sources) on the TensorFlow site for your platform, up to the end of 'Configure the installation'.
 
-To build the TensorFlow library, execute the following command:
+Then you can build the Tensorflow and DeepSpeech libraries.
 
 ```
-bazel build -c opt //tensorflow:libtensorflow.so
-```
-
-Then you can build the DeepSpeech native library.
-
-```
-bazel build -c opt //native_client:*
+bazel build -c opt --copt=march=native --copt=mtune=native --copt=-O3 //tensorflow:libtensorflow.so //native_client:deepspeech
```

Comment:
nit: isn't `//native_client:deepspeech` going to build only the deepspeech client binary, and not the depending libdeepspeech.so ?